### ISSUE TITLE
Move "q" and "sorby" to features specification

### ISSUE THREAD

**Issue by pvretano** [opengeospatial/ogcapi-records#126]
28-JUN-2021: At the TC there was discussion about moving all the API aspects of record to features.  Basically q and sortby would be moved to features since they are relevant there too.  Thus the RECORDS API specification would more-or-less be about the information model (i.e. the record).  The API would be completely encapsulated in features.

**Comment by jyutzler** [comment]
This issue is closely related to #125. If this standard is solely about the information model, then we should rename it - it is no longer an Implementation Standard (like OGC API-Features), but rather an Encoding Standard (like GeoPackage). I would be fine with moving all of the API-based stuff to Features (or confirm it is there) as long as the Features SWG members concur.

**Comment by pvretano** [comment]
04-OCT-2021: This is has not been discussed in Features so we will put it on the Features SWG agenda for next week.  We also need to review with @ogcscotts what this means with respect to the charter.

**Comment by ogcscotts** [comment]
@pvretano The Charter still covers the case of where Records is evolving - SWGs are often about discovery and new ideas! No need to update the Charter.

**Comment by pvretano** [comment]
29-NOV-2021: We don't have quorum to make the official decision but no one on the call objects to having features take 1 and sortby AND to have records simply say "implement features as the search API".

**Comment by pvretano** [comment]
04-APR-2022:  Features is just finishing up CQL2 and Part 3 but once that is done they will circle back to q and sortby.  In the meantime we will continue to develop these building blocks in Records.

**Comment by kalxas** [comment]
Reading https://github.com/opengeospatial/ogcapi-features/issues/636 seems like Features is waiting for our agreement to move forward...

**Comment by pvretano** [comment]
@kalxas that comment was published in 2021 ... the Records SWG has long since given its concent ... both @cportele and I are aware of this and will move `q` and `sortby` in due time.

**Comment by kalxas** [comment]
Yes, I was referring to how it looks like, not how it actually is :)

**Comment by pvretano** [comment]
08-JUL-2024:  Features has created two parts, 8-sorting and 9-text search, to initiate the process of moving these into features.  It is unlikely that Features will be done before OAPIR becomes adopted so we will leave q and sortby in and in a future version we can remove these sections and normatively reference Parts 8 & 9 from Features.
